flowauth.Dockerfile

```diff
@@ -3,7 +3,6 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 FROM node:16 as builder
-# Node version pinned until https://github.com/nodejs/docker-node/issues/1379 is closed
```

Comment:
Why deleting the comment but not the pin?

Reply:
Because _another_ issue just appeared... 
https://github.com/eslint/eslint-scope/issues/55